bokeh/server/server.py

```diff
@@ -27,9 +27,10 @@ def _create_hosts_whitelist(host_list, port):
     hosts = []
     for host in host_list:
         if '*' in host:
-            log.warning('Host wildcard %r can expose the application to HTTP '
-                        'host header attacks. Host wildcard should only be '
-                        'used for testing purpose.', host)
+            log.warning(
+                "Host wildcard %r will allow websocket connections originating "
+                "from multiple (or possibly all) hostnames or IPs. Use non-wildcard "
+                "values to restrict access explicitly", host)
```

Comment:
Wondering about making this a `log.info`, perhaps even making it `'*'` by default. 

@havocp I understand why `--host` was previously needed when we were relying on the HOST header. Is there some reason for keeping allowed ws origins tightened down by default?